src/test/java/uk/gov/hmcts/darts/arm/service/CleanupArmResponseFilesServiceImplTest.java

```diff
@@ -0,0 +1,139 @@
+import uk.gov.hmcts.darts.common.entity.ExternalLocationTypeEntity;
+import uk.gov.hmcts.darts.common.entity.ExternalObjectDirectoryEntity;
+import uk.gov.hmcts.darts.common.entity.MediaEntity;
+import uk.gov.hmcts.darts.common.entity.ObjectRecordStatusEntity;
+import uk.gov.hmcts.darts.common.entity.UserAccountEntity;
+import uk.gov.hmcts.darts.common.helper.CurrentTimeHelper;
+import uk.gov.hmcts.darts.common.repository.ExternalLocationTypeRepository;
+import uk.gov.hmcts.darts.common.repository.ExternalObjectDirectoryRepository;
+import uk.gov.hmcts.darts.common.repository.ObjectRecordStatusRepository;
+
+import java.time.OffsetDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyBoolean;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class CleanupArmResponseFilesServiceImplTest {
+
+    @Mock
+    private ExternalObjectDirectoryRepository externalObjectDirectoryRepository;
+    @Mock
+    private ObjectRecordStatusRepository objectRecordStatusRepository;
+    @Mock
+    private ExternalLocationTypeRepository externalLocationTypeRepository;
+    @Mock
+    private ArmDataManagementApi armDataManagementApi;
+    @Mock
+    private UserIdentity userIdentity;
+    @Mock
+    private ArmDataManagementConfiguration armDataManagementConfiguration;
+    @Mock
+    private CurrentTimeHelper currentTimeHelper;
+
+    @Mock
+    private ExternalLocationTypeEntity externalLocationTypeArm;
+    @Mock
+    private ObjectRecordStatusEntity objectRecordStatusStored;
+    @Mock
+    private ObjectRecordStatusEntity objectRecordStatusArmResponseChecksumFailed;
+    @Mock
+    private ObjectRecordStatusEntity objectRecordStatusArmResponseProcessingFailed;
+    @Mock
+    private ObjectRecordStatusEntity objectRecordStatusArmResponseManifestFailed;
+    @Mock
+    private ExternalObjectDirectoryEntity externalObjectDirectory;
+
+    @Mock
+    private MediaEntity media;
+    @Mock
+    private UserAccountEntity testUser;
+
+    private CleanupArmResponseFilesService cleanupArmResponseFilesService;
+
+    @Captor
+    private ArgumentCaptor<ExternalObjectDirectoryEntity> externalObjectDirectoryEntityCaptor;
+
+
+    @BeforeEach
+    void setUp() {
+        cleanupArmResponseFilesService = new CleanupArmResponseFilesServiceImpl(
+            externalObjectDirectoryRepository,
+            objectRecordStatusRepository,
+            externalLocationTypeRepository,
+            armDataManagementApi,
+            userIdentity,
+            armDataManagementConfiguration,
+            currentTimeHelper
+        );
+    }
+
+    @Test
+    void cleanupResponseFilesSuccess() {
+
+        when(externalLocationTypeRepository.getReferenceById(3)).thenReturn(externalLocationTypeArm);
```

Comment:
I'd prefer to see us using the standing data enum here in the tests for better readability like we use them in our service code.